classes/submission/reviewer/ReviewerSubmissionDAO.inc.php

```diff
@@ -45,7 +45,33 @@ function getReviewerSubmission($reviewId) {
 		$locale = AppLocale::getLocale();
 		$result = $this->retrieve(
 			'SELECT	m.*, p.date_published,
-				r.*,
+				r.review_id,
```

Comment:
Can you add the code comment here as well?

Reply:
Sure - Done